app/models/base_model.rb

```diff
@@ -14,4 +14,8 @@ def update_file!(name)
     raw_write_attribute(:filename, name)
     save!
   end
+
+  def public_stories(arel = nil)
```

Comment:
we need the method in series, network, account, and playlist (I think), which seemed like enough to justify putting it in there, but you know, maybe I should make a simple concern instead...